client/modules/App/components/Overlay.jsx

```diff
@@ -68,17 +69,22 @@ class Overlay extends React.Component {
           >
             <header className="overlay__header">
               <h2 className="overlay__title">{title}</h2>
-              <div className="overlay__actions">
-                {actions}
-                <button
-                  className="overlay__close-button"
-                  onClick={this.close}
-                  aria-label={this.props.t('Overlay.AriaLabel', { title })}
-                >
-                  <ExitIcon focusable="false" aria-hidden="true" />
-                </button>
-              </div>
+              <MediaQuery query="(min-width: 768px)">
+                <div className="overlay__actions">{actions}</div>
+              </MediaQuery>
+              <button
+                className="overlay__close-button"
+                onClick={this.close}
+                aria-label={this.props.t('Overlay.AriaLabel', { title })}
+              >
+                <ExitIcon focusable="false" aria-hidden="true" />
+              </button>
```

Comment:
The `button` used to be inside the `overlay__actions` div and now it's outside.  As a result the search box is now more center-aligned, as they are now separate items inside the flex parent.
![image](https://github.com/processing/p5.js-web-editor/assets/28965286/7bf91a59-afb2-4d95-8633-fcf9afee4967)
Can we keep it the same as how it was before?
![image](https://github.com/processing/p5.js-web-editor/assets/28965286/7a53ebf9-2014-4f3d-9b46-00700c4c7402)
```suggestion
              <div className="overlay__actions">
                <MediaQuery query="(min-width: 768px)">{actions}</MediaQuery>
                <button
                  className="overlay__close-button"
                  onClick={this.close}
                  aria-label={this.props.t('Overlay.AriaLabel', { title })}
                >
                  <ExitIcon focusable="false" aria-hidden="true" />
                </button>
              </div>
```